test/functional/combine_logs.py

```diff
@@ -6,11 +6,11 @@
 
 import argparse
 from collections import defaultdict, namedtuple
-import glob
 import heapq
 import os
 import re
 import sys
+from test_framework.util import MAX_NODES
```

Comment:
micronit: blank line between the standard library imports and local project imports please.